src/organisms/Navbar/Navbar.stories.tsx

```diff
@@ -41,14 +41,40 @@ Default.args = {};
 
 export const WithLinks = Template.bind({});
 WithLinks.args = {
-    links: ["Link 1", "Link 2", "Link 3"],
-    activeLink: "Link 1",
+    links: [
+        {
+            href: "https://www.google.com",
+            label: "Google",
+        },
+        {
+            href: "https://www.facebook.com",
+            label: "Facebook",
+        },
+        {
+            href: "https://www.twitter.com",
+            label: "Twitter",
+        },
+    ],
+    activeLink: "https://www.twitter.com",
```

Comment:
does it make more sense to check on the label? Its much of a muchness but I think the assumption here is that there will be neither
- two links with the same label
- two links with the same href
and since labels are generally shorter and more readable could just check the label